doc/release-tasklist

```diff
@@ -57,13 +57,24 @@
       cd doc && ./bump_version.sh $TAG && cd .. && \
       git commit -m "version $TAG" && \
       git tag -s v$TAG -m "version $TAG"
-  . create a tar file:
+  . create a tar file: TODO: remove manual.pdf
```

Comment:
What about this TODO?

Reply:
I thought I don't spend the time right now to change and test the commands to not include the manual.pdf in the tarball but do this next time. Do you want me to update the bash commands now?

Reply:
That would be great, yes.